common/lib/xmodule/xmodule/x_module.py

```diff
@@ -385,9 +387,9 @@ def get_content_titles(self):
                          SEQUENCE
                             |
                          VERTICAL
-                        /        \
+                        /        \\
```

Comment:
We're really getting into the nits now, but I would use vertical pipes instead of messing around with backslashes, or make the docstring an r"""...""" string.